### DISCUSSION TITLE
Disable error squiggles while I'm coding

### DISCUSSION THREAD

**Discussion by ThomasMoncrief** [microsoft/pylance-release#3337] [Enhancement]
Hi,

I have been searching for over an hour, and I am quite confident that there is no option to disable to squiggles that pop up as I'm coding. Am I blind as to how to disable this feature, or is it really impossible? Either way, this needs to be addressed. Either make the setting easier to change or put it in. I don't want squiggles to distract me as I'm altering code. How are you not being bombarded with requests for this?

I'm disabling your extension until I can figure out how to manage something so simple as to not have distracting squiggles as I work on something. It's really a shame, because I like being able to have the summary of problems in the 'Problems' tab below. But I'm not going to enable the extension if it means squiggles flashing on and off constantly. This is such a simple request. How is this not a setting?

**Comment by judej** [comment]
Thanks for the suggestion. I will move this to discussions for upvotes.

**Comment by ccrvlh** [comment]
Maybe adding an option to set a delay could be useful, like: `wait X ms before adding the squiggles` `squiggles delay time`

**Comment by rchiodo** [comment]
VS code has kind of a hacky way to do it:

https://stackoverflow.com/questions/43454967/disable-wavy-underline-in-vs-code

Essentially turn the color off. Problems will still show in the problems tab though.
